components/search/loader.js

```diff
@@ -1,35 +1,46 @@
 import React from 'react'
+import ContentLoader from 'react-content-loader'
+import { theme } from 'ooni-components'
 
-import styled from 'styled-components'
-
-import {
-  Flex, Box,
-  Heading
-} from 'ooni-components'
-
-import SpinLoader from '../vendor/spin-loader'
-
-const StyledLoader = styled.div`
-  width: 100%;
-  padding-bottom: 100px;
-  display: ${props => props.loading === true ? 'block' : 'none'};
-`
-
-const Loader = ({loading}) => (
-  <StyledLoader loading={loading}>
-    <Flex alignItems='baseline' justifyContent='space-around'>
-      <Box>
-        <Flex flexDirection='column'>
-          <Box style={{height: '200px'}}>
-            <SpinLoader />
-          </Box>
-          <Box style={{textAlign: 'center'}}>
-            <Heading h={3}>Loading</Heading>
-          </Box>
-        </Flex>
-      </Box>
-    </Flex>
-  </StyledLoader>
+export const Loader = () => (
+  <ContentLoader
+    height={450}
+    width={450}
+    speed={1}
+    primaryColor={theme.colors.gray3}
+    secondaryColor={theme.colors.gray5}
+  >
+    <circle cx="20" cy="33" r="5" />
+    <rect x="40" y="30" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="30" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="30" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="63" r="5" />
+    <rect x="40" y="60" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="60" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="60" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="93" r="5" />
+    <rect x="40" y="90" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="90" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="90" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="123" r="5" />
+    <rect x="40" y="120" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="120" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="120" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="153" r="5" />
+    <rect x="40" y="150" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="150" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="150" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="183" r="5" />
+    <rect x="40" y="180" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="180" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="180" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="213" r="5" />
+    <rect x="40" y="210" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="210" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="210" rx="3" ry="3" width="70" height="5" />
+    <circle cx="20" cy="243" r="5" />
+    <rect x="40" y="240" rx="3" ry="3" width="100" height="5" />
+    <rect x="160" y="240" rx="3" ry="3" width="70" height="5" />
+    <rect x="250" y="240" rx="3" ry="3" width="70" height="5" />
```

Comment:
Maybe the loader svgs should go in a specific svg file so they can be easily updated and modified by designers?

Reply:
Actually I see that it's not really a simple svg. I will push some changes to this directly on the branch.